akka-remote-tests/src/multi-jvm/scala/akka/remote/artery/AeronStreamLatencySpec.scala

```
@@ -129,6 +130,7 @@ abstract class AeronStreamLatencySpec
     taskRunner.stop()
     aeron.close()
     driver.close()
+    IoUtil.unmap(cncByteBuffer)
```

let it crash perhaps then 😉